### PR TITLE
Add server date to devicelog report

### PR DESCRIPTION
**Pull request by proteusvacuum**
http://manage.dimagi.com/default.asp?227465#1149703

Adds server date to the devicelog report, otherwise they are somewhat confusing as the 60 day purge acts on the `server_date`

### CODE REVIEW

**Comment by proteusvacuum** [comment]
@nickpell @benrudolph 
cc @dannyroberts

File: corehq/ex-submodules/phonelog/reports.py [hunk 1]
```diff
@@ -73,7 +73,10 @@ class BaseDeviceLogReport(GetParamsMixin, DatespanMixin, PaginatedReportMixin):
     @property
     def headers(self):
         return DataTablesHeader(
-            DataTablesColumn("Date", span=1, sort_type=DATE, prop_name='date',
+            DataTablesColumn("Log Date", span=1, sort_type=DATE, prop_name='date',
```
Comment (czue) [inline corehq/ex-submodules/phonelog/reports.py]: not your fault originally but these should probably be marked for translation

File: corehq/ex-submodules/phonelog/reports.py [hunk 2]
```diff
@@ -73,7 +73,10 @@ class BaseDeviceLogReport(GetParamsMixin, DatespanMixin, PaginatedReportMixin):
...
+                             sort_direction=[DTSortDirection.DSC,
+                                             DTSortDirection.ASC]),
+            DataTablesColumn("Server Date", span=1, sort_type=DATE, prop_name='server_date',
```
Comment (czue) [inline corehq/ex-submodules/phonelog/reports.py]: I could see this terminology being a little confusing. Could maybe change to Log Date/Log Submission Date to match some of our other reports?

**Comment by czue** [comment]
no blockers :+1: on :white_check_mark:

**Pull request merged**